local-tests/setup/tinny-environment.ts

```diff
@@ -381,7 +381,7 @@ export class TinnyEnvironment {
         dAppOwnerWallet: wallet,
         capacityTokenId: capacityTokenId,
         // Sets a maximum limit of 200 times that the delegation can be used and prevents usage beyond it
```

Comment:
```suggestion
        // Sets a maximum limit of 2000 times that the delegation can be used and prevents usage beyond it
```

Reply:
Ah I have some left over changes I should remove these.